dojo/dojo

```diff
@@ -73,6 +73,7 @@ case "$ACTION" in
         multi_node=$(jq 'length > 0' /data/workspace_nodes.json 2>/dev/null || echo "unknown")
         if [ "$WORKSPACE_NODE" -gt 0 ]; then
             profiles+=("workspace")
+            profiles+=("workspace_exclusive")
```

Comment:
On production migration, we need to make sure the new nginx service starts up, and then also that port 80 and 443 are open.